.gitignore

```diff
@@ -1,5 +1,4 @@
 /vendor
-.env
```

Comment:
I think this should still be ignored to prevent people committing their secrets?

Reply:
Yep, it's just duplicated (line 29)